api/app/Models/User.php

```diff
@@ -833,43 +833,6 @@ public function setRoleAssignmentsInputAttribute($roleAssignmentHasMany)
         }
     }
 
-    // reattach all the extra fields from the JSON data column
-    public static function enrichNotification(object $notification)
```

Comment:
Don't we still want this enrich function? Maybe in Notification.php's boot function?

Reply:
It is, it just isn't as explicit so I can add some comments. It uses the magic `getAttribute` method to enrich it.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/c7559b23cff31b3032e9b0c480e152e105aa63ee/api/app/Models/Notification.php#L16-L25

Reply:
😮